openlibrary/plugins/openlibrary/api.py

```diff
@@ -3,12 +3,104 @@
 import simplejson
 
 from infogami.utils import delegate
-from openlibrary.core import helpers as h
+from infogami.utils.view import render_template
+from openlibrary.core import lending, helpers as h
+from openlibrary.data import popular
+
+popular_editions = popular.popular
+
 
 def setup():
     # placeholder
     pass
 
+class popular_books(delegate.page):
+    path = '/popular'
+
+    def GET(self):
+        """Returns `limit` popular book tuples of form [ocaid, olid] starting
+        at `start`
+
+        Should be cached through memcached.
+        """
+        i = web.input(start='0', limit='100')
+        start, limit = int(i.start), int(i.limit),
+        books = popular_editions[start:start+limit]
```

Comment:
Following https://www.python.org/dev/peps/pep-0008/#id26, adding spacing though not required around '+'